packages/stark-core/src/modules/http/services/http.service.spec.ts

```diff
@@ -1098,9 +1093,18 @@ describe("Service: StarkHttpService", () => {
 					item: mockResourceWithoutEtag,
 					serializer: mockResourceSerializer
 				};
-				const result: ErrorObservable = <ErrorObservable>starkHttpService.executeSingleItemRequest(request);
-				expect(result instanceof ErrorObservable).toBe(true);
-				expect(result.error).toContain("Unknown request type");
+
+				starkHttpService.executeSingleItemRequest(request).subscribe(
+					() => {
+						fail("The 'next' function should not be called in case of error");
+					},
+					(error: any) => {
```

Comment:
I think the type of error is string in this case...

Reply:
done